cwms-data-api/src/test/java/cwms/cda/data/dto/LocationLevelTest.java

```diff
@@ -1,84 +1,82 @@
-				.withOfficeId("SPK")
-				.build();
+		ZonedDateTime zdt = ZonedDateTime.parse("2021-06-21T08:00:00-07:00[PST8PDT]");
+		String tsId = "Test.Elev.Ave.1Day.Regulating";
+		final TimeSeriesLocationLevel level = new TimeSeriesLocationLevel.Builder("Test", zdt, tsId).build();
 
-		assertThrows(RequiredFieldException.class, () -> {
-			noParameter.validate();
-		});
+		ContentType contentType = Formats.parseHeader(Formats.JSONV2, LocationLevel.class);
+		String jsonStr = Formats.format(contentType, level);
 
+		// If JSONv2 isn't setup correctly it will serialize the level like:
+//		{"location-level-id":"Test","level-date":1624287600.000000000}
+
+		assertTrue(jsonStr.contains("2021"));
+	}
 
-		final LocationLevel constAndSeasonalId = new LocationLevel.Builder("Test",ZonedDateTime.now())
-													.withOfficeId("SPK")
-													.withConstantValue(5.0)
-													.withSeasonalTimeSeriesId("The test timeseries")
-													.build();
+	@Test
+	void test_serialization_formats_Constant()
+	{
+		ZonedDateTime zdt = ZonedDateTime.parse("2021-06-21T08:00:00-07:00[PST8PDT]");
+		final ConstantLocationLevel level = new ConstantLocationLevel.Builder("Test", zdt).build();
 
-		assertThrows(ExclusiveFieldsException.class, () -> {
-			constAndSeasonalId.validate();
-		});
+		ContentType contentType = Formats.parseHeader(Formats.JSONV2, LocationLevel.class);
+		String jsonStr = Formats.format(contentType, level);
 
+		assertTrue(jsonStr.contains("2021"));
+	}
 
-		// we don't need actual values for this test, just the system to think there might be
-		final LocationLevel constAndValBean = new LocationLevel.Builder("Test",ZonedDateTime.now())
-													.withOfficeId("SPK")
-													.withConstantValue(5.0)
-													.withSeasonalValues(new ArrayList<>())
-													.build();
-		assertThrows(ExclusiveFieldsException.class, () -> {
-			constAndSeasonalId.validate();
-		});
+	@Test
+	void test_serialization_formats_Seasonal()
+	{
+		ZonedDateTime zdt = ZonedDateTime.parse("2021-06-21T08:00:00-07:00[PST8PDT]");
+		final SeasonalLocationLevel level = new SeasonalLocationLevel.Builder("Test", zdt).build();
 
+		ContentType contentType = Formats.parseHeader(Formats.JSONV2, LocationLevel.class);
+		String jsonStr = Formats.format(contentType, level);
 
+		assertTrue(jsonStr.contains("2021"));
 	}
 
 	@Test
-	void test_serialization_formats()
+	void test_serialization_formats_Virtual()
 	{
 		ZonedDateTime zdt = ZonedDateTime.parse("2021-06-21T08:00:00-07:00[PST8PDT]");
-		final LocationLevel level = new LocationLevel.Builder("Test", zdt).build();
+		final VirtualLocationLevel level = new VirtualLocationLevel.Builder("Test", zdt).build();
 
 		ContentType contentType = Formats.parseHeader(Formats.JSONV2, LocationLevel.class);
 		String jsonStr = Formats.format(contentType, level);
 
-		// If JSONv2 isn't setup correctly it will serialize the level like:
```

Comment:
even though these comments are duplicate, they should be retained, and probably even added to the one above.

anyone of these tests should be reasonable in isolation so it is reasonable to see duplication we would otherwise avoid in normal code.